newsroom/reports/__init__.py

```diff
@@ -40,4 +36,6 @@ def init_app(app):
     superdesk.register_resource("news_api_search", WireSearchResource, WireSearchService, _app=app)
 
 
-from . import views  # noqa
+from .views import blueprint  # noqa
+
+module = Module(name="newsroom.reports", endpoints=[blueprint])  # type: ignore
```

Comment:
What is the reason with `# type: ignore`, what is causing the type to be invalid?